.bazelrc

```diff
@@ -1,3 +0,0 @@
-common --experimental_enable_bzlmod
-build --incompatible_enable_cc_toolchain_resolution
-build --incompatible_strict_action_env
```

Comment:
deleting this last line does affect things, the first 2 probably don't matter if you updated bazel, since they both default to true as of bazel 7.x.

based on the warning you posted below you just needed to remove `experimental_` from the first one to solve that warning

Reply:
the `.bazelrc` doesn't affect upstream users of pcre2 so it shouldn't matter since we're only testing against 8.x anyways

Reply:
yea, but this option does apply to that case. the last flag here reduces the chances of the local env affecting the build for example